brianlong/www.validators.app
# 187231219 process vote transactions

#### What's this PR do?
- add vote transactions processing to get block script

#### How should this be manually tested?
- run migrations
- run tests
- make sure You have current EpochWallClock and slots (`rails r script/blockchain/get_leaders_schedule.rb`)
- run sidekiq with blockchain queue
- run `rails r daemons/blockchain/slot_subscribe_mainnet_daemon.rb`
- there should be new Blockchain::Transactions created 

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187231219)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
